content/v1.12/concepts/managed-resources.md

```diff
@@ -3,630 +3,745 @@ title: Managed Resources
-> they have is currently undocumented. This is tracked in [this
-> issue][issue-1143].
 
-#### Immutable Properties
+### Pause
+Manually applying the `crossplane.io/paused` annotation causes the Provider to
+stop reconciling the managed resource. 
 
-There are configuration parameters in external resources that cloud providers do
-not allow to be changed. For example, in AWS, you cannot change the region of an
-`RDSInstance`.
+Pausing a resource is useful when modifying Providers or preventing
+race-conditions when editing Kubernetes objects.
 
-Some infrastructure tools such as Terraform delete and recreate the resource to
-accommodate those changes but Crossplane does not take that route. Unless the
-managed resource is deleted and its `deletionPolicy` is `Delete`, its controller
-never deletes the external resource in the provider.
+Apply a {{<hover label="pause" line="6">}}crossplane.io/paused: "true"{{</hover>}}
+annotation to a managed resource to pause reconciliation. 
 
-> Kubernetes does not yet support immutable fields for custom resources. This
-> means Crossplane will allow immutable fields to be changed, but will not
-> actually make the desired change. This is tracked in [this issue][issue-727].
+{{< hint "note" >}}
+Only the value `"true"` pauses reconciliation.
+{{< /hint >}}
 
-#### Pausing Reconciliations
-If a managed resource being reconciled by the [managed reconciler], has the
-`crossplane.io/paused` annotation with its value set to `true` as in the
-following example, then further reconciliations are paused on that resource
-after emitting an event with the type `Synced`, the status `False`,
-and the reason `ReconcilePaused`:
-```yaml
+```yaml {label="pause"}
 apiVersion: ec2.aws.upbound.io/v1beta1
-kind: VPC
+kind: Instance
 metadata:
-  name: paused-vpc
+  name: my-rds-instance
   annotations:
     crossplane.io/paused: "true"
-...
+spec:
+  forProvider:
+    region: us-west-1
+    instanceType: t2.micro
 ```
-Reconciliations on the managed resource will resume once the 
-`crossplane.io/paused` annotation is removed or its value is set
-to anything other than `true`.
 
-### External Name
+Remove the annotation to resume reconciliation.
 
-By default the name of the managed resource is used as the name of the external
-cloud resource that will show up in your cloud console. To specify a different
-external name, Crossplane has a special annotation to represent the name of the
-external resource. For example, I would like to have a `CloudSQLInstance` with
-an external name that is different than its managed resource name:
+## Finalizers
+Crossplane applies a 
+[Finalizer](https://kubernetes.io/docs/concepts/overview/working-with-objects/finalizers/)
+on managed resources to control their deletion. 
+
+{{< hint "note" >}}
+Kubernetes can't delete objects with Finalizers.
+{{</hint >}}
+
+When a Crossplane deletes managed resource the Provider begins deleting the
```

Comment:
Should this read instead "When Crossplane starts to delete a managed resources, the Provider begins..."